### PR TITLE
Fix formatting of the example custom action script

### PR DESCRIPTION
**Pull request by vpodzime**

### CODE REVIEW

**Comment by vpodzime** [comment]
@cf-bottom can you help me with jenkins here, please?

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/571/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/571/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-571/output/_site/

File: web-ui/custom-actions-for-alerts.markdown [hunk 1]
```diff
@@ -104,8 +104,8 @@ When you get this to work as expected on the commmand line, you are ready to upl
 ## Example script: logging policy alert to syslog
 
 The following Custom action script will log the status and definition of a policy alert to syslog.
+
 ```bash
-[file=cfengine_custom_notification_policy_syslog.sh]
```
Comment (olehermanse) [inline web-ui/custom-actions-for-alerts.markdown:108]: This line is intentional / should be there to create the file components.
But looks like the site generation is buggy, so your commit does improve the end result. Let's merge it and I'll try to figure out what the real bug is here 😅

**Review by olehermanse — APPROVED** [review]

File: web-ui/custom-actions-for-alerts.markdown (hunk 1 above)
Comment (olehermanse) [reply web-ui/custom-actions-for-alerts.markdown:108]: Yep, this line was not the problem; https://github.com/cfengine/documentation/pull/3259

**Comment by olehermanse** [comment]
Cherry-pick: https://github.com/cfengine/documentation/pull/3260#issuecomment-2049768635

**Pull request merged**